string ID
int8 status
